plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClientImpl.kt

```diff
@@ -0,0 +1,64 @@
+// Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.amazonq.lsp
+
+import com.intellij.notification.NotificationType
+import org.eclipse.lsp4j.ConfigurationParams
+import org.eclipse.lsp4j.MessageActionItem
+import org.eclipse.lsp4j.MessageParams
+import org.eclipse.lsp4j.MessageType
+import org.eclipse.lsp4j.PublishDiagnosticsParams
+import org.eclipse.lsp4j.ShowMessageRequestParams
+import software.aws.toolkits.jetbrains.services.amazonq.lsp.model.aws.credentials.ConnectionMetadata
+import software.aws.toolkits.jetbrains.services.amazonq.lsp.model.aws.credentials.SsoProfileData
+import java.util.concurrent.CompletableFuture
+
+/**
+ * Concrete implementation of [AmazonQLanguageClient] to handle messages sent from server
+ */
+class AmazonQLanguageClientImpl : AmazonQLanguageClient {
+    override fun telemetryEvent(`object`: Any) {
+        println(`object`)
```

Comment:
nit: not needed?

Reply:
we're going to keep the prints until they are implemented properly